demo/screens/Home.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Image, Button, StyleSheet, View, Text } from "react-native";
+import { createStackNavigator, createAppContainer } from "react-navigation";
+import { Account } from "tasit-sdk";
+import ListLands from "./ListLands";
+import LandClaim from "./LandClaim";
+import OnboardingHome from "./OnboardingHome";
+import EthereumQuestion from "./EthereumQuestion";
+import EthereumSignUp from "./EthereumSignUp";
+import EthereumSignIn from "./EthereumSignIn";
+
+class Home extends React.Component {
+  componentDidMount = async () => {
+    let wallet = await Account.create();
+    console.log(wallet.address);
+  };
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <Image source={require("../assets/images/icon.png")} />
+        <Text style={styles.tasitText}>
+          {`Tasit
+          `}
+        </Text>
+        <Button
+          title="Decentraland"
+          onPress={() => this.props.navigation.navigate("ListLands")}
+        />
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: "#fff",
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  tasitText: {
+    fontSize: 20,
+    textAlign: "center",
+    fontWeight: "bold",
+    color: "gray",
+  },
+});
+
+const AppNavigator = createStackNavigator(
```

Comment:
I feel like creating this navigator may belong in a different file. I'll keep thinking on this.

Reply:
Yeah, the home component itself should be in a different file from the AppNavigator file

Reply:
Please tell me if I've done right.

Reply:
Yeah, I like the change you made!